hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/state/virtual/UniqueTokenKeySerializerTest.java

```diff
@@ -47,13 +47,12 @@ void deserializeToUniqueTokenKey_whenValidVersion_shouldMatch() throws IOExcepti
 
     @Test
     void serializeUniqueTokenKey_shouldReturnExpectedBytes() throws IOException {
-        final ByteArrayOutputStream byteStream = new ByteArrayOutputStream();
+        final ByteBuffer byteBuffer = ByteBuffer.allocate(17);
```

Comment:
Might improve clarity to introduce a constant for the buffer length, so it's a little easier to work out where `17` comes from (2 long values and a single byte).